client/js/main.ts

```diff
@@ -369,27 +414,81 @@ document.getElementById("add-attendee")!.addEventListener("click", e => {
 	});
 });
 
+// Add tags to users
+document.getElementById("add-new-tag")!.addEventListener("click", e => {
+	let button = (<HTMLButtonElement> e.target)!;
```

Comment:
`let button = e.target as HTMLButtonElement;` would be a simpler way to express this